server/server.js

```diff
@@ -68,6 +69,20 @@ app.get(`${apiBaseUrl}/participants`,
   asyncMiddleware(async (req, res) => {
     const user = req.hcapUserInfo;
     const result = await getParticipants(user);
+
+    // HCAP-242 - Log View Access Of Participant Data
+    const { content } = req.kauth.grant.access_token;
+    const logData = {
+      "action": "participant_get",
+      "accessed_by": {
+        "username": content.preferred_username,
+        "id": content.sub
+      },
+      //slicing so that things don't get out of hand with the IDs
```

Comment:
I would probably tweak this comment - the important thing to note here is that we picked 10 to log because those are the records that come back on the first page of results :)